WeScanTests/VisionRectangleDetectorTests.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  VisionRectangleDetectorTests.swift
+//  WeScanTests
+//
+//  Created by James Campbell on 8/8/18.
+//  Copyright © 2018 WeTransfer. All rights reserved.
+//
+
+import XCTest
+
+class VisionRectangleDetectorTests: XCTestCase {
```

Comment:
This file is not doing anything?

Reply:
I need some advice on how to tackle this one and the CIRectangle detector, I guess they need full integration testing with an image we've created passed into the detector?

Reply:
We could try to detect the rectangle inside this image, and check if the detected rectangle is close enough to the preset rectangle we code in.